config.ts

```diff
@@ -0,0 +1,4 @@
+//Add all the env variables in this file.
+
```

Comment:
Create a type definition for the type of object we'll receive from this config file. It'll be easier for us to handle the exports this way.

Reply:
Also, create an object instead of directly attaching individual variables to the exports object. Assign the object at the end as default.

```ts

interface someType {
  discordBotToken: string,
}

var exportObject: someType = {
  discordBotToken: process.env.DISCORD_BOT_TOKEN,
};
export default exportObject;
```

Like this example. You can change this to a way that suits you but make sure it's exported by default since we'll only be having this object from this file.